invenio_records_resources/records/systemfields/files/manager.py

```diff
@@ -198,18 +197,21 @@ def commit(self, file_key):
         self[file_key] = file_obj
 
     @ensure_enabled
-    def delete(self, key, remove_obj=True, softdelete_obj=False):
-        """Delete a file."""
+    def delete(self, key, remove_obj=True, softdelete_obj=True, remove_rf=False):
+        """Delete a file.
+
+        Defaults to soft deletion of record file metadata and object versions.
```

Comment:
minor: I would add docstrings for the arguments since we are using abbreviations in the arguments' names, it may help to understand the code faster in the future